R/cn.mops.R

```diff
@@ -564,18 +568,37 @@ cn.mops <- function(input,I = c(0.025,0.5,1,1.5,2,2.5,3,3.5,4),
 			colnames(callsS) <- colnames(X)
 			for (chrom in chrOrder){
 				chrIdx <- chrDf[chrom,1]:chrDf[chrom,2]
+				
 				if (parallel==0){
-					resSegmList[[chrom]] <- apply(sINI[chrIdx, ,drop=FALSE],2,
+					if(moderate_amplifications){
+						resSegmList[[chrom]] <- apply(sINI[chrIdx, ,drop=FALSE],2,
+							segment,
+							minSeg=minWidth,
+							alpha=0.065,
+							segMedianT=c(0.25,-0.5),...)	
+					}
+					else {
+						resSegmList[[chrom]] <- apply(sINI[chrIdx, ,drop=FALSE],2,
 							segment,
 							minSeg=minWidth,...)
+					}
 				} else {
-					cl <- parallel::makeCluster(as.integer(parallel),type="SOCK")
-					parallel::clusterEvalQ(cl,"segment")
-					resSegmList[[chrom]] <- parallel::parApply(cl,sINI[chrIdx, ,drop=FALSE],2,
-							segment,minSeg=minWidth,...)
-					parallel::stopCluster(cl)
+					
+					if(moderate_amplifications){
+						cl <- parallel::makeCluster(as.integer(parallel),type="SOCK")
+						parallel::clusterEvalQ(cl,"segment")
+						resSegmList[[chrom]] <- parallel::parApply(cl,sINI[chrIdx, ,drop=FALSE],2,
+								segment,minSeg=minWidth,alpha=0.065,segMedianT=c(0.25,-0.5),...)
```

Comment:
is the change to alpha really necessary?

Reply:
yes. this is necessary for this line: https://github.com/Ultimagen/cn.mops/blob/ba697e55bca62962c3ad12f588d2cca7beaa793e/R/segment.R#L111 
to lower down the cutoff to consider segments as potential CNVs.